src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -187,6 +187,40 @@
                 </attribute>
             </itemOperation>
 
+            <itemOperation name="shop_account_change_payment_method">
```

Comment:
It is necessary?  why don't you use here `shop_select_payment_method`? Now you are duplicating path configuration only for adding new operation name

Reply:
I think that we need have it on `/account` path

Reply:
If we've introduced the second endpoint, won't it make sense, to separate the behavior of a handler?

Reply:
@lchrusciel You are right, but its 100% the same behavior(without guard for `/account`), and it increases maintenance for the next service. And if you change some thing in one, you have to remember to change it in next

Reply:
@Tomanhez only if you duplicate service, IMHO after @lchrusciel suggestion you should exclude some logic to new service and use it in two handlers. for example service `PaymentMethodSetter` and `PaymentMethodChanger` or some better names :D